# Determination - Deterministic rendering environment for white-axe's music
# Copyright (C) 2024 Liu Hao <whiteaxe@tuta.io>
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.

# This patch disables a log message that appears when a ZynAddSubFX automation
# slot binding is successfully created.

--- a/src/Misc/Master.cpp
+++ b/src/Misc/Master.cpp
@@ -739,7 +739,7 @@ void Master::loadAutomation(XMLwrapper &xml, rtosc::AutomationMgr &midi)
                                 offset = atof(node["offset"].c_str());
                             }
                         }
-                        printf("createBinding(%d, %s, false)\n", i, path.c_str());
+                        //printf("createBinding(%d, %s, false)\n", i, path.c_str());
                         midi.createBinding(i, path.c_str(), false);
                         midi.setSlotSubGain(i, j, gain);
                         midi.setSlotSubOffset(i, j, offset);
